easybuild/easyconfigs/p/Perl/Perl-5.30.0-GCCcore-8.3.0.eb

```diff
@@ -1637,11 +1637,36 @@ exts_list = [
         'source_urls': ['https://cpan.metacpan.org/authors/id/R/RR/RRA'],
         'checksums': ['7af1c41de34b2e4dbff700a29d7387549c2b6cf16142214450c924707ddb0f82'],
     }),
+    ('LWP::UserAgent', '6.43', {
```

Comment:
This is actually already installed, via ``LWP::Simple::` (same source tarball `libwww-perl`...)

Reply:
fixed in ec1dcf6ef4